### DISCUSSION TITLE
Mac 打开下载的文件打不开www

### DISCUSSION THREAD

**Discussion by Yinun0609** [natfrp/wiki#232] [使用问题] [answered]
下载后没后缀，就是“frpc_darwin_amd64”
双击打开后就显示
“未能打开文稿“frpc_darwin_amd64”。文本编码Unicode (UTF-8)不适用。”
跪求大佬帮帮忙啊 小白不会搞

**Comment by fakeboboliu** [comment, marked as answer]
你好，你在帮助文档 repo 里，请看帮助文档

**Reply by Yinun0609** [reply]
谢谢！

**Reply by MYF114514** [reply]
我也是这个问题，但是帮助文档里面没有找到解决办法，是我找错了吗？

**Reply by Derrickzz** [reply]
下载后没后缀，就是“frpc_darwin_amd64”
双击打开后就显示
“未能打开文稿“frpc_darwin_amd64”。文本编码Unicode (UTF-8)不适用。”

**Comment by Yinun0609** [comment]
我已经找到解决方法了！
https://doc.natfrp.com/frpc/usage.html
打开连接后选择macos，根据教程安装！



在 2022-12-27 14:16:27，"MYF114514" ***@***.***> 写道：

我也是这个问题，但是帮助文档里面没有找到解决办法，是我找错了吗？

**Comment by Yinun0609** [comment]
是的，直接打开是无法运行的。你需要依照我上一个邮件中给你的链接在终端中进行安装操作，这很详细！



在 2023-01-05 22:22:00，"Derrickzz" ***@***.***> 写道：

下载后没后缀，就是“frpc_darwin_amd64”
双击打开后就显示
“未能打开文稿“frpc_darwin_amd64”。文本编码Unicode (UTF-8)不适用。”
